ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -83,16 +83,21 @@ sub Configure_Cache {
     my $file_cache      = 0;     # how much cache is actually configured now in the config file
     my $file_cachemount = '';    # where should the cache be mounted per config file
     my $new_cache       = 0;     # in 1k blocks.
-    my $file_afsmount   = '';
+    my $file_afsmount   = '/afs';
```

Comment:
ok, i sort of understand, this is for the empty file case.
better to use add the default values as readonly on top the module, and do
```perl
use Readonly;
Readonly my $DEFAULT_AFSMOUNT => '/afs';

...
my $file_afsmount   = $DEFAULT_AFSMOUNT;
```

and probably similar for file_cachemount and new_cache.

```